lib/utils.pm

```diff
@@ -1329,7 +1329,7 @@ can be set to 1.
 =cut
 sub reconnect_mgmt_console {
     my (%args) = @_;
-    $args{timeout}             //= 300;
+    $args{timeout}             //= 500;
```

Comment:
Quick question, from the VRs and the ticket it's not clear how this timeout bump helps. In the failed run `reconnect_mgmt_console` appears to wrk fine (https://openqa.opensuse.org/tests/1615154#step/reconnect_mgmt_console/8). Maybe you can propose a change, so that this module fails in such cases, so failure is more clear, as `chown` command didn't put result on serial.

Reply:
You are right, that reconnect_mgmt_console has got no issues. We have got a test issue that we are running into a timeout in line 1185 in the file utils.pm. Firstly, I looked into system_prepare.pm. Changing the timeout here would not have any effect because that counts only for SLE.
I looked, how and where a timeout is set in the file utils.pm because it used as a base by system_prepare.pm and here is the issue. I thought about why we are running in this timeout and where it could be set. I found the definition of args with timeout in reconnect_mgmt_console. system_prepare is using a console, too. Therefore I wanted to test the effect of expanding this timeout, whether this value will be imported to system_prepare.pm via utils as a variable for the whole file. I changed the value and executed a verification run. We don't have any failed results in the verification run any more.

Reply:
Alright, I took some time to take a closer looks and seems I've identified the root cause. It still feels like this change didn't really help, as we did not spend 300 seconds in any of the modules.
You can see times next to the modules in the run view:
![image](https://user-images.githubusercontent.com/28305643/106769711-8a0bcf00-663d-11eb-9af0-e3aee1beb320.png)

Also last run did work fine: https://openqa.opensuse.org/tests/1616040

So I've described my finding in #11892 If that helps in VR (I guess I should trigger more runs to prove that it helps), but seems that issue is that we match wrong string and leave the module too early, and we fail to grab from serial. I might be wrong with this theory, but that should still help to stabilize execution TW tests on zVMs.

Reply:
Interesting! You have changed the $login_ready in the same function I identified for the reason. But I would not have the idea to make the change to get_login_mesage() as an openQA beginner. ^^
I wanted to expand the timeout as a solution.
Thank you for the reference!

Reply:
Yeah, unfortunately many parts of openQA are obscure and not well documented. I've merged #11892. Let's see  if that helps to stabilize test execution and dig further if not ;)